client/driver/rkt.go

```diff
@@ -188,20 +205,34 @@ func (d *RktDriver) Start(ctx *ExecContext, task *structs.Task) (DriverHandle, e
 	// Add CPU isolator
 	cmdArgs = append(cmdArgs, fmt.Sprintf("--cpu=%vm", int64(task.Resources.CPU)))
 
-	// Add DNS servers
-	for _, ip := range driverConfig.DNSServers {
-		if err := net.ParseIP(ip); err == nil {
-			msg := fmt.Errorf("invalid ip address for container dns server %q", ip)
-			d.logger.Printf("[DEBUG] driver.rkt: %v", msg)
-			return nil, msg
-		} else {
-			cmdArgs = append(cmdArgs, fmt.Sprintf("--dns=%s", ip))
-		}
+	// Fingerprint
+	node := &structs.Node{
+		Attributes: make(map[string]string),
 	}
+	_, err := d.Fingerprint(&config.Config{}, node)
```

Comment:
@diptanu I was trying to get rkt version by doing `d.config.Node.Attributes["driver.rkt.version"]`, but that gave me a `invalid memory address or nil pointer dereference`. While the `Fingerprint` function sets the version key, I noticed that the `Attributes` map did not have that key in `Start`. Was trying to work around that problem here. Any idea why it crashes?